docs/docs/user/references/components.md

```diff
@@ -1,8 +1,16 @@
 # Components
 
 This section explains the different components of KPOps, their usage and
-configuration.
-<!-- To learn more about their hierarchy, visit the
+configuration via [`pipeline.yaml`](/resources/pipeline-components/pipeline).
+
+--8<--
+./docs/resources/architecture/components-hierarchy.md
+--8<--
+
+See real-world examples of `pipeline.yaml` [here](/resources/examples/pipeline).
```

Comment:
if possible, consider putting the link behind `pipeline.yaml` directly for a better user experience. :) in other words, generally we'd like to avoid adding the word "here" just for the link

Reply:
Good point! I think it's better to make the whole sentence a link though. `pipeline.yaml` already links to the examples page earlier in the document and it would be a bit confusing if it also links to somewhere else.